src/styles/index.scss

```
@@ -1,3 +1,133 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+p {
+  margin: 0;
+}
+
+$accent-color: #00acdc;
+$main-color: #060b35;
+$second-color: #616070;
+$card-width: 200px;
+
+.container {
+  position: relative;
+}
+
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  * {
+    box-sizing: border-box;
+  }
```

Don't need to duplicate the properties, you can put the ``*`` tag on top, which will set this style for the entire page.

```suggestion

* {
    box-sizing: border-box;
}

.container {
  position: relative;
}

.card {
  width: $card-width;
  padding: 32px 16px 16px;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
```